pkgs/os-specific/linux/pam_gnupg/default.nix

```
@@ -0,0 +1,29 @@
+{stdenv, fetchurl, autoreconfHook, libtool, pam, gnupg }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  pname = "pam_gnupg";
+  version = "0.1";
+
+  src = fetchurl {
+    url = "https://github.com/cruegge/pam-gnupg/archive/v${version}.tar.gz";
+    sha256 = "1i4xllwkfpby97jgha1qv8s6crlxiacjagm3gx3fvc2z30ca1qdn";
+  };
+
+  configureFlags = [
+    "--with-moduledir=${placeholder "out"}/lib/security"
+  ];
+
+  buildInputs = [ pam gnupg ];
+
+  nativeBuildInputs = [ autoreconfHook libtool ];
+
+  meta = {
+    homepage = https://github.com/cruegge/pam-gnupg;
+    description = "Unlock GnuPG keys on login";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ 0x6d6178 ];
```

Oh and it seems there's a problem with your username :)
```suggestion
    maintainers = with maintainers; [ vxid ];
```

---

My bad. I am about to change Github accounts so I put my new identity there.